tests/test_minimizer.py

```diff
@@ -197,3 +199,44 @@ def test_local_minimize_water_box():
     u_init_test, g_init_test = val_and_grad_fn(x0)
     assert u_init == u_init_test
     np.testing.assert_array_equal(g_init, g_init_test)
+
+
+@pytest.mark.xfail(reason="Doesn't work currently, but should eventually")
+def test_minimizer_failure_toy_system():
+    """https://github.com/proteneer/timemachine/pull/1346 introduced a minimization scheme that minimizes
+    at lambda = 0.1 and at 0.0. The need for lambda = 0.1 was due to minimizing PFKFB3 with ligands
+    20 and 43, this test replicates the failure case with a small toy system to allow investigation in the future
+    """
+
+    # Atoms 2933 and 38815 from PFKFB3 with ligands 20, 43
+    nb_params = np.array(
+        [
+            [0.9323587, 0.13247664, 0.25629826, 0.0],
+            [0.52629348, 0.12916129, 0.26202344, 0.0],
+        ]
+    )
+    beta = 2.0
+    cutoff = 1.2
+    pairlist = NonbondedPairList([0, 1], [1.0, 1.0], beta, cutoff)
+
+    coords = np.array(
+        [
+            [9.6643, 5.5546, 26.9625],
+            [9.65109, 5.51557, 26.93242],
+        ]
+    )
```

Comment:
coords can be simplified / made less pfkfb3-testsystem-specific

```python
dx = 0.04  # anywhere between 1e-20 and 0.04 seems to produce the same initial force norm

coords = np.array(
	[
		[0,  0,  0],
		[dx, dx, dx],
	]
)
```

Reply:
Made less specific in https://github.com/proteneer/timemachine/pull/1351/commits/5c8a33798ec73a392ec9d1770fd2981c5881d2ae